playground/src/i18n/data/core.json

```diff
@@ -1,24 +1,19 @@
 {
-  "Menu.StateMachines": "State Machines",
-  "Menu.EchoServer": "Echo Server",
```

Comment:
These messages are used for the menu links, please keep them (see `playground/custom-application-config.mjs`)

Reply:
Yes, thanks for catching this! [4d13a0d](https://github.com/commercetools/merchant-center-application-kit/pull/3403/commits/4d13a0dd4ba0251a9b04dcecb2e23be46f36da18)